presto-native-execution/presto_cpp/main/PrestoTask.cpp

```diff
@@ -775,27 +763,6 @@ void PrestoTask::updateExecutionInfoLocked(
       prestoTaskStats);
 }
 
-/*static*/ std::string PrestoTask::taskNumbersToString(
```

Comment:
We might want to keep this as static method of PrestoTask as the states are specific to PrestoTask internal

s/taskNumbersToString/taskStatesToString/